angular-spa-ui/js/search-app/tags-module/tags-ctrl.js

```diff
@@ -1,75 +1,10 @@
 "use strict";
 
-module.exports = function(tags) {
+module.exports =  function mainTagCtrl($scope, resultTagPageSrv) {
```

Comment:
module.exports is an anonymous function, it could not have a name. Remove 'mainTagCtrl'.